README.md

```diff
@@ -152,6 +152,54 @@ Nap is customized through environment variables:
   />
 </p>
 
+## Snippet Management from Terminal
```

Comment:
Hey, thanks for this PR, I think we can simplify a lot of these functions to simply be aliases:

```
alias napclip="nap $(nap list | gum filter) | xclip -selection clipboard"
alias napexec="nap $(nap list | gum filter) | bash"
alias napsave="gum write | nap $(gum input --placeholder folder/snippet.ext)"
```